### PR TITLE
fix(input): resolve RTL support (#2315)

### LINKED ISSUE #2315: [BUG] - Lack of RTL Support in Input
### NextUI Version

2.0.10

### Describe the bug

The repository does not work correctly when the direction is set to RTL (Right-to-Left). Elements are not displayed as expected, leading to usability issues.

### The label is not in the right place
[See Here](https://storybook.nextui.org/?path=/story/components-input--label-placement&globals=locale:ar-AE)

image

### The Clear Button is not in the right place
[See Here](https://storybook.nextui.org/?path=/story/components-input--clearable&globals=locale:ar-AE)

image

### Your Example Website or App

_No response_

### Steps to Reproduce the Bug or Issue

1. Go to [https://storybook.nextui.org/?path=/story/components-input--label-placement&globals=locale:ar-AE]
2. Change the document direction to RTL (locale: Arabic).
3. Observe the issues mentioned above.

### Expected behavior

I expected the UI to properly support RTL direction, with elements aligning and displaying correctly from right to left.

### Screenshots or Videos

_No response_

### Operating System Version

All

### Browser

Chrome

### PR DESCRIPTION
**Pull request by mrbadri**
Closes #2315

## 📝 Description

Add RTL support to the input component.

## ⛳️ Current behavior (updates)

The input component currently lacks support for right-to-left (RTL) direction.

## 🚀 New behavior

This PR addresses the RTL support issue in the input component, ensuring correct rendering in RTL environments.

## 📝 Additional Information

- Resolves issue #2315
- Added RTL support to the input component

### CODE REVIEW

**Comment by jrgarciadev** [comment]
@mrbadri it worked fine, thank you!, could you please add the changeset?

**Comment by mrbadri** [comment]
Hi [jrgarciadev](https://github.com/jrgarciadev),  thank you for review my pull request

when i run 'pnpm changeset'
i get this error : 
🦋  Which packages would you like to include? · No items were selected
🦋  error You must select at least one package to release
🦋  error (You most likely hit enter instead of space!)

i can't add changeset

**Comment by mrbadri** [comment]
Hi [jrgarciadev](https://github.com/jrgarciadev),
I add changeset

File: .changeset/curly-pandas-guess.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/input": major
```
Comment (jrgarciadev) [inline .changeset/curly-pandas-guess.md:2]: @mrbadri 

```suggestion
"@nextui-org/input": patch
```

Update the changeset to be a `patch`, when creating it just press enter without selecting that it is a `major` / `minor`

**Review by jrgarciadev — CHANGES_REQUESTED** [review]

**Comment by itmysm** [comment]
Hi Mohammad Reza! if you need any help with RTL support, I'm ready to help you.

**Comment by mrbadri** [comment]
Hi [jrgarciadev](https://github.com/jrgarciadev),

I hope you're doing well! I've made some updates to address the feedback you provided earlier. Could you please take another look at my pull request when you have a moment? Your feedback is greatly appreciated.

Thank you!

Mohammad Reza Badri

**Review by jrgarciadev — APPROVED** [review]
Thank you! @mrbadri

**Pull request merged**